### PR TITLE
Childminder Applicant / Adult disq

### PR DESCRIPTION
**Pull request by witoldpiecewicz**
## Description

Childminder Applicant
Childminder Adult

Finished all the tasks for Suitability questions:
- views
- templates
- forms
- tests
- updated task_list

## Todo's before PR

- [ ] Branch has been rebased against develop
- [ ] Unit tests passed (`make test`)
- [ ] PR named appropriately - see [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Commits-guideline)
- [ ] PR description describes the overall goals of PR commits
- [ ] Templates have been checked against the relevant user-story

## Migrations

- [ ] Null fields in models specifies default value
- [ ] You generated and applied migrations (`make migrate`)
- [ ] You updated fixtures (`make export`)

## PR Todo's

Please refer to PR [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-submit-a-pull-request)

- [ ] Specified reviewers (even if it's yourself)
- [ ] Specified assignees (those who'll merge)
- [ ] Specified labels

## PR review

Please refer to PR review [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-review-a-pull_request)

- [ ] Code syntax formatting checked
- [ ] Debug messages are absent

## PR merge

Select only one:

- [ ] Should be merged?
- [ ] Should be rebased?
- [ ] Should be squashed?

Please refer to PR merge [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-merge-a-pull_request)

### CODE REVIEW

File: application/models/suitability.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    social_services = models.NullBooleanField(
+        blank=True, null=True, default=None)
+    social_services_details = models.TextField(max_length=1000, null=True)
+    criminal_convictions = models.NullBooleanField(
+        blank=True, null=True, default=None)
+    criminal_convictions_details = models.TextField(max_length=1000, null=True)
+    criminal_convictions = models.NullBooleanField(
+        blank=True, null=True, default=None)
+    criminal_convictions_details = models.TextField(max_length=1000, null=True)
+
```
Comment (jenniferbrown94) [inline application/models/suitability.py:31]: Can you add in the timelog fields method? You'll see this in other models, every field except the application_id should be included in the timelog

**Review by jenniferbrown94 — CHANGES_REQUESTED** [review]

File: application/models/suitability.py (hunk 1 above)
Comment (witoldpiecewicz) [reply application/models/suitability.py:28]: Done

**Comment by jenniferbrown94** [comment]
I'm happy with this PR now - I will approve once we are ready to merge

**Comment by jenniferbrown94** [comment]
Hi @witoldpiecewicz, @nitz14 - there is an item in the https://jira.informed.com/browse/ODQ-42 to remove the existing social services question - this is at the end of the people in the home section and should be removed from the journey and any summary pages.

**Comment by witoldpiecewicz** [comment]
@jenniferbrown94 it has been removed

**Comment by jenniferbrown94** [comment]
rerun jenkins build

**Comment by witoldpiecewicz** [comment]
@jenniferbrown94 the issues with visual alignment have been resolved

**Comment by jenniferbrown94** [comment]
rerun jenkins build

**Comment by jenniferbrown94** [comment]
rerun jenkins build

**Review by jenniferbrown94 — APPROVED** [review]

**Pull request merged**